search.go

```diff
@@ -261,6 +261,7 @@ func (h *HighlightRequest) AddField(field string) {
 // Explain triggers inclusion of additional search
 // result score explanations.
 // Sort describes the desired order for the results to be returned.
+// Score controls the kind of scoring performed
```

Comment:
May do good to document here or else where, that accepted parameters for score today are "none", "tf/idf", ""?

Reply:
yah, I was a bit hesitant on this, as the proposed PR currently supports only "" and "none".

The "none" turns off scoring.

The "" gives the current behavior as a default.  But, I wasn't sure what to explicitly name the current behavior!  "tf/idf" scoring?  "basic" scoring?  "regular" scoring?  "default" scoring?  hmm.